website/static/js/osfHelpers.js

```diff
@@ -191,6 +191,19 @@ var errorDefaultShort = 'Unable to resolve';
 var errorDefaultLong = 'OSF was unable to resolve your request. If this issue persists, ' +
     'please report it to <a href="mailto:support@osf.io">support@osf.io</a>.';
 
+var handleHTTPError = function(error){
```

Comment:
This name can be more specific, since it is only handling the revoke api loading error.